<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorWidget</class>
 <widget class="QWidget" name="MonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>监视空调</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="11">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>服务器参数</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Modelabel">
     <property name="text">
      <string>模    式</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mode_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>制热</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="3">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="FeeLlabel">
     <property name="text">
      <string>费    率</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="FeeLEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>元/度</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" rowspan="3">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="7" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>风速</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>低</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QLineEdit" name="FanLEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0.33</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>度/分钟</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="DefaultFanlabel">
     <property name="text">
      <string>默认风速</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="default_fan_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>中</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="TempLowlabel">
     <property name="text">
      <string>最低温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="TempLowEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>18</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>中</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QLineEdit" name="FanMEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>度/分钟</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="DefaultTemplabel">
     <property name="text">
      <string>默认温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="DefaultTempEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>22</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="TempHighlabel">
     <property name="text">
      <string>最高温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLineEdit" name="TempHighEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>高</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QLineEdit" name="FanHEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>度/分钟</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="11">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="11">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>空调状态</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="11">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>880</width>
        <height>289</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="RoomBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="8" column="0" colspan="11">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>880</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="MonitorBacktoWidgetButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
